### PR TITLE
To Evacuate or Not to Evacuate.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/Open To Evacuate or Not to Evacuate.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+8. At the start of the game, we selected one subject (the ‘informant’) at random who was informed in advance whether a disaster would indeed strike or not. The behaviours of the informants did not vary significantly across the network treatments
+>  The exact sentence that the informants received in their game screen was ‘A disaster is going to strike!’ when a disaster would strike or ‘There is no disaster’. when a disaster would not strike. Uninformed subjects received notification that ‘A disaster may or may not strike’.
+9. The other subjects were informed that some players had accurate information about the disaster, and they were also told that immediate neighbours of the informant would know the identity of the informant by our marking ‘i’ on the informant's node in their network diagram in the game screen. When a disaster struck before they evacuated, the subjects earned no bonus. Otherwise, they earned a bonus of $2.00 without the disaster or $1.00 with the disaster by spending $1.00 to evacuate, plus $0.10 per other player who took the correct action accordingly
```
Comment (katelynbottcher) [inline tasks/Open To Evacuate or Not to Evacuate.md:24]: In the network conditions, subjects shared their information; when clicking the 'Safe' button, their node turned blue and after 5 seconds automatically turned to grey, and likewise, when clicking the 'Danger' button, their node turned red. Subjects could only see the colors of neighbors that they were directly connected. Once subjects chose to evacuate, they couldn't send signals and their node turned grey, and their neighbors weren't informed of their evacuation.

File: tasks/Open To Evacuate or Not to Evacuate.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Roles 
+The Informant: informed in advance whether a disaster would indeed strike or not
+
+## Instructions
+Reached out to instructors 5/5
+
+# Criteria
+## Performance calculation
+A formal specification of how performance is evaluated based on the specific materials and the teams actions in a given trial.
```
Comment (katelynbottcher) [inline tasks/Open To Evacuate or Not to Evacuate.md:35]: Performance was calculated based on the participant's ability to correctly avoid an impending disaster, and the payoffs are displayed in the Stimuli sections

File: tasks/Open To Evacuate or Not to Evacuate.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# References
+Main Paper: https://royalsocietypublishing.org/doi/full/10.1098/rspa.2019.0685#d1e503
+Data Files: https://humannaturelab.net/publications/collective-communication-and-behaviour-in-response-to-uncertain-danger-in-network-experiments
+
+# Stimuli
+## The visual components
+![image](https://user-images.githubusercontent.com/78745728/117169986-9b2cbf00-ad97-11eb-84d7-66849fa6f64d.png)
+
+
```
Comment (katelynbottcher) [inline tasks/Open To Evacuate or Not to Evacuate.md:12]: Network topology varied as either a random-regular network or a ring-lattice network. Network size was also manipulated to have 10, 20, 40, or 60 nodes.

**Review by sarikasu — COMMENTED** [review]
added instructions from researchers

**Review by katelynbottcher — APPROVED** [review]

**Pull request merged**